house/urls.py

```diff
@@ -9,4 +9,5 @@
     path('house/', views.house_view, name='house_view'),
     path('house/add_expense/', views.add_expense, name='add_expense'),
     path('ajax/load-cities/', views.load_cities, name='ajax_load_cities'),
+    path('house/edit_house/<int:id>/', views.edit_house, name='edit_house'),
```

Comment:
please use the user object from the request to get the house id
This way other users cant use this API and it will be more secure

Reply:
fixed